py/gc.c

```diff
@@ -236,7 +218,21 @@ STATIC void gc_drain_stack(void) {
         void **ptrs = (void**)PTR_FROM_BLOCK(block);
         for (size_t i = n_blocks * BYTES_PER_BLOCK / sizeof(void*); i > 0; i--, ptrs++) {
             void *ptr = *ptrs;
-            VERIFY_MARK_AND_PUSH(ptr);
+            if (VERIFY_PTR(ptr)) {
+                // Mark and push this pointer
+                // TODO: almost-duplicate code in gc_collect_root
```

Comment:
I don't think these TODO's should stay because there's nothing "to do": it's not really possible to refactor this (or part of this) code into a common macro.  I'd be happy not to have any comment about the almost-duplicated code.

Reply:
Fixed.